.github/workflows/go.yml

```diff
@@ -3,7 +3,11 @@ on: [push]
 jobs:
   build:
     name: Build
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04, ubuntu-20.04, ubuntu-latest, macos-latest, windows-latest]
```

Comment:
either add a mac-specific file or remove from here. And for ubuntu I think we can leave just `ubuntu-latest`